app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class RentsController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    rent = user.rents.new(rent_params)
+
+    if rent.save
+      render json: rent, status: :created, serializer: RentSerializer
```

Comment:
By default rails searches a serializer with the same model name, remove the serializer option